src/pages/api/tap/cmac.ts

```diff
@@ -135,7 +135,7 @@ export default async function handler(
     },
   });
   if (location) {
-    const message = uuidv4();
+    const message = uuidv4().replace(/-/g, ""); // Message should be a hex string
```

Comment:
Why don't we also do nonces here, just for consistency? Can also display what # you were while tapping

Reply:
Great idea! https://github.com/nfc-denver/nfc-denver/issues/122